File: 3483-unique-3-digit-even-numbers/3483-unique-3-digit-even-numbers.js
/**
 * @param {number[]} digits
 * @return {number}
 */
var totalNumbers = function(digits) {
    
    let numbers = new Set();
    for(let i = 0; i < digits.length; i++) {
        for(let j = 0; j < digits.length; j++) {
            for(let k = 0; k < digits.length; k++) {
                if (i != k && j != k && i != j) {
                    let number = digits[i] * 100 + digits[j] * 10 + digits[k]
                    if (number > 99 && number % 2 == 0) {
                        numbers.add(number);
                    }
                }
            }
        }
    }
    return numbers.size;
};

File: 3483-unique-3-digit-even-numbers/3483-unique-3-digit-even-numbers.py
class Solution:
    def totalNumbers(self, digits: List[int]) -> int:
        
        numbers = set()
        for i in range(len(digits)):
            for j in range(len(digits)):
                for k in range(len(digits)):
                    if i != j and j != k and i != k:
                        number = digits[i] * 100 + digits[j] * 10 + digits[k]
                        if number % 2 == 0 and number > 99:
                            numbers.add(number)
        return len(numbers)